consensus/state.go

```diff
@@ -162,9 +156,6 @@ type State struct {
 	propagator           propagation.Propagator
 	newHeightOrRoundChan chan struct{}
 
-	// nextBlock contains the next block to propose when building blocks during the timeout commit
-	nextBlock chan *blockWithParts
```

Comment:
Is this part of reverting the pipelined block building? Ideally that should be a separate PR

Reply:
I'm fine with both. Want me to separate this into a separate PR?